example/android/build.gradle

```diff
@@ -14,17 +14,18 @@ buildscript {
 allprojects {
     repositories {
         google()
-        if (project.hasProperty('onegini_artifactory_user') && project.hasProperty('onegini_artifactory_password')) {
+        def (artifactoryUser, artifactoryPassword) = getArtifactoryCredentials()
+        if (artifactoryUser?.trim() && artifactoryPassword?.trim() && artifactoryUser != "" && artifactoryPassword != "") {

```

Comment:
Trim returns a new string with white spaces removed it does not mutate the string, is that what you're trying to check here? https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/trim.html
If you are trying to trim them, please do it in the helper function.

Reply:
will pull from develop where this is fixed